### PR TITLE
Adds a more thorough example of transforming content.

### PR DESCRIPTION
**Pull request by manovotny**

### CODE REVIEW

File: docs/advanced/transform-content.md [hunk 1]
```diff
@@ -26,15 +46,44 @@ const mdx = require('remark-mdx');
...
+    { type: "jsx", value: "<Component />", position: [Object] }
+  ],
+  position: {
+    start: { line: 1, column: 1, offset: 0 },
+    end: { line: 12, column: 1, offset: 113 }
+  }
+}
+```
+
+From here, you can continue to [write your own plugin][writing-a-plugin] or use any of the [remark plugins][remark-plugins] to interpolate MDX content.
```
Comment (wooorm) [inline docs/advanced/transform-content.md]: Is `interpolate` the correct term here? English isn’t my primary language, so maybe it is, but it seems a bit jargon-y to me. Maybe just drop the ` to interpolate MDX content` part?

Comment (manovotny) [reply docs/advanced/transform-content.md]: @wooorm I went with `parse or manipulate MDX content` instead.

Comment (wooorm) [reply docs/advanced/transform-content.md]: Good good!

**Comment by johno** [comment]
Thanks!

**Pull request merged**